ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -64,6 +65,11 @@ const mapStateToProps = (state, ownProps) => {
     }
   }
 
+  const isSnap = Boolean(
+    targetSubjectMetadata &&
+      targetSubjectMetadata.subjectType === SUBJECT_TYPES.SNAP,
```

Comment:
Nit: We can do `targetSubjectMetadata?.subjectType === SUBJECT_TYPES.SNAP` and remove the `targetSubjectMetadata &&`